db/system_keyspace.cc

```diff
@@ -2223,8 +2226,9 @@ future<> system_keyspace::get_compaction_history(compaction_history_consumer con
 }
 
 future<> system_keyspace::update_repair_history(repair_history_entry entry) {
-    sstring req = format("INSERT INTO system.{} (table_uuid, repair_time, repair_uuid, keyspace_name, table_name, range_start, range_end) VALUES (?, ?, ?, ?, ?, ?, ?)", REPAIR_HISTORY);
+    sstring req = format("INSERT INTO system.{} (table_uuid, repair_time, repair_uuid, keyspace_name, table_name, range_start, range_end) VALUES (?, ?, ?, ?, ?, ?, ?)  USING TTL {}", REPAIR_HISTORY, s_repair_histroy_ttl_seconds);
```

Comment:
No need for the static variable. We can get the ttl directly from cfg on the fly.  This makes sure the new value will be used.